pilot/pkg/model/service.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	endpoint "github.com/envoyproxy/go-control-plane/envoy/config/endpoint/v3"
 	"github.com/mitchellh/copystructure"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
this adds a new dependency on the previously k8s agnostic `model` package. I am surprised this doesn't actually fail a test, I would have thought it would bloat the pilot-agent binary but maybe not. Regardless, it would be good to absrtact this; we can just copy the 4 strings into this package instead of import